README.md

```diff
@@ -156,25 +160,25 @@ pyenv global 3.10.4   # switches the global interpreter to 3.10.4
 
 You can verify this works as expected using `python --version`. You may be familiar with using `python3` as the command instead of `python`. With `pyenv`, this is not necessary.
 
-Additional usage and installation instructions can be found in the [pyenv repo](https://github.com/pyenv/pyenv). 
+Additional usage and installation instructions can be found in the [pyenv repo](https://github.com/pyenv/pyenv).
 
 ## Installing `poetry` for dependency resolution and publishing packages
 
 Reference: [Poetry docs](https://python-poetry.org/docs/)
 
-Poetry can be installed with both `curl` and `pip`. We recommended using `curl` so that it will be global to your machine. 
+Poetry can be installed with both `curl` and `pip`. We recommended using `curl` so that it will be global to your machine.
 
-NOTE We highly, highly, highly recommend that you DO NOT use `brew` to install `poetry`. 
+NOTE We highly, highly, highly recommend that you DO NOT use `brew` to install `poetry`.
 If you use `brew`, it's going to install directly to your system, which prevents you from being able to leverage `pyenv` to seamlessly switch between Python interpreters.
 
-```sh 
+```sh
 # installation with pip: recommended option in tandem with pyenv
 pip install poetry
 ```
 
 ```sh
 # For UNIX systems - installation with curl 
-curl -sSL https://install.python-poetry.org/ | python -
+curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
 ```
```

Comment:
That URL has a story. In [this installation section](https://python-poetry.org/docs/master/#installing-with-the-official-installer) of the poetry site, it says the following:
> The installer script is available directly at [install.python-poetry.org](https://install.python-poetry.org/), and is developed in [its own repository](https://github.com/python-poetry/install.python-poetry.org). The script can be executed directly (i.e. ‘curl python’) or downloaded and then executed from disk (e.g. in a CI environment).
![image](https://user-images.githubusercontent.com/51418232/184464511-973daba3-f4ae-4eb3-b6b4-27d9b2b27b78.png)

They then recommend for Linux, macOS, and WSL to use
```sh
curl -sSL https://install.python-poetry.org | python3 -
```

Reply:
Btw, how are you linting the markdown files?